Test/git-issues/git-issue-2500.dfy

```diff
@@ -0,0 +1,22 @@
+// RUN: %dafny_0 /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module A {
+  trait {:termination false} Trait {
+    function TotallyNotZero() : (ret: int)
```

Comment:
Also add some tests that use the function itself (rather than the named result value `ret`), since those cases may be tricky.